gina.txt

```diff
@@ -0,0 +1 @@
+my file
```

Comment:
if you make the fine `gina.md` (or some other name that ends in `.md`, and add [Markdown](https://daringfireball.net/projects/markdown/syntax#overview) markup, it will be all styled.